reference-implementation/lib/transform-stream.js

```diff
@@ -121,8 +123,8 @@ function TransformStreamErrorInternal(transformStream, e) {
 
   transformStream._chunk = undefined;
 
-  if (transformStream._resolveWriter !== undefined) {
-    transformStream._resolveWriter(undefined);
+  if (transformStream._rejectWriter !== undefined) {
+    transformStream._rejectWriter(e);
```

Comment:
s/rejectWriter/rejectWrite/

anyway, I just noticed that this is unnecessary since we call _writableController.error() at L118. I'm going to just remove this in #533